ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -347,12 +367,13 @@ private static void putEdges(
     int heightIndex = IS_JELLY_BEAN_OR_LATER ? photos.getColumnIndex(Images.Media.HEIGHT) : -1;
     int longitudeIndex = photos.getColumnIndex(Images.Media.LONGITUDE);
     int latitudeIndex = photos.getColumnIndex(Images.Media.LATITUDE);
+    int dataIndex = photos.getColumnIndex(MediaStore.MediaColumns.DATA);
```

Comment:
I tried to limit the amount of changes to functional parts only. There are quite a few things that can be improved but I am just afraid to expand the scope of this PR too much.

dataIndex is needed for extracting the uri of the file. I found this the easiest way to reliably obtain the file uri.